Week-1/Assignment1.2/index.html

```diff
@@ -0,0 +1,72 @@
+    <div style="position: absolute; padding-left: 280px; margin-top: -25px;">
+    	<i class="fa-solid fa-play" style="color:white;"><t style="color:white; font-size: 12px; padding-left: 8px;">Play all</t></i>
+    	<i class="fa-solid fa-plus" style="color:white; padding-left: 20px;"><t style="color:white; font-size: 12px; padding-left: 8px;">Add to</t></i>
+    	<i class="fa-solid fa-ellipsis" style="color:white; padding-left: 20px;"><t style="color:white; font-size: 12px; padding-left: 8px;">More</t></i>
+    </div>
+  </header>
+  	<body>
+		<body id="ibw9">
+  			<div id="ihzf">
+   			 	<table class="tg" id="i0f1">
+      				<tbody>
+        				<tr>
+          					<td class="tg-cly1">1. One Dance</td>
+          					<td class="tg-0lax">Crake feat CatKid &amp; Cyla</td>
+          					<td class="tg-0lax">2:54</td>
+          					<td class="tg-7h26"><a href="https://www.google.com/" target="_blank" rel="noopener noreferrer">CATTY CLOUD SYNC</a></td>
+        				</tr>
+        				<tr>
+          					<td class="tg-buh4">2. Panda</td>
+          					<td class="tg-buh4">Cattee</td>
+          					<td class="tg-buh4">4:06</td>
+          					<td class="tg-buh4"><a href="https://google.com/" target="_blank" rel="noopener noreferrer">CATTY CLOUD SYNC</a></td>
+        				</tr>
+        				<tr>
+          					<td class="tg-0lax">3. Can&#039;t Stop the Feeling!</td>
+          					<td class="tg-0lax">Catin Cimberlake</td>
+          					<td class="tg-0lax">3:56</td>
+          					<td class="tg-0lax"><a href="https://google.com/" target="_blank" rel="noopener noreferrer">CATTY CLOUD SYNC</a></td>
+        				</tr>
+        				<tr>
+          					<td class="tg-buh4">4. Work From Home</td>
+          					<td class="tg-buh4" id="i1i5j">Cat Harmony feat Colla</td>
+          					<td class="tg-buh4">3:34</td>
+          					<td class="tg-yjjc"><a href="https://google.com/" target="_blank" rel="noopener noreferrer">CATTY CLOUD SYNC</a></td>
+        				</tr>
+      				</tbody>
+    			</table>
+  			</div>
+  				<div class="row" id="igqj1">
+  					<div style="padding-left: 600px; padding-top: 40px; position: absolute;">
+  					<i class="fa-solid fa-backward-step" style="color:white;"></i>
+  					<i class="fa-solid fa-pause" style="padding-left: 100px; color: white;"></i>
+  					<i class="fa-solid fa-forward-step" style="padding-left: 100px; color: white;"></i>
+  					<i class="fa-solid fa-rotate-right" style="padding-left: 100px; color: white;"></i>
+  					<i class="fa-solid fa-shuffle" style="padding-left: 100px; color: white;"></i>
+  					</div>
+    				<div class="cell">
+    					<img id="i8ydg" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"/>
+    					<div style="position: absolute; margin-top: -100px; padding-left: 110px;"><t style="color:white; font-size: 12px; padding-left: 8px;">If It Ain't Love</t>
```

Comment:
what does this tag `<t> </t>` do ?

Reply:
only to enhance the text, removed as it didnt have much importance